gap/Utils.gi

```diff
@@ -594,7 +594,85 @@ function(epsilon, d, q)
         D[m + 1, m] := zeta * (xi + xi ^ q) ^ (-1);
     fi;
 
-    return rec(generatorsOfOmega := generatorsOfOmega, S := S, G := G, D := D);        
+    return rec(generatorsOfOmega := generatorsOfOmega, S := S, G := G, D := D);
+end);
+
+# Construct standard generators L1, L2, L3 as used in [HR10]
+# with the following properties similar to Theorem 3.11 in [HR10]:
+#   * L1 and L2 generate GL(d, q)
+#   * L1 and L3 generate SL(d, q)
+#   * all matrix entries lie in {0, \pm 1, \pm zeta^{\pm 1}}, where
+#       zeta is a primitive element of GF(q)
+#   * If q is odd, L1 and L2^2 generate the unique subgroup
+#       of index 2 of GL(d, q), often denoted (1 / 2) GL(d, q).
+# Construction as in [T87]
+BindGlobal("StandardGeneratorsOfLinearGroup",
+function(d, q)
+    local field, one, zeta, L1, L2, L3;
+
+    field := GF(q);
+    one := One(field);
+    zeta := PrimitiveElement(field);
+
+    if d = 1 then
+
+        L1 := [[one]];
+        L2 := [[zeta]];
+        L3 := [[one]];
+
+    elif d = 2 and q = 3 then
+
+        L1 := [[-one, one], [-one, 0 * one]];
+        L2 := [[-one, one], [one, 0 * one]];
```

Comment:
a matter of taste, but I usually just write it like this (seems more readable to me)
```suggestion
        L1 := one * [[-1, 1], [-1, 0]];
        L2 := one * [[-1, 1], [1, 0]];
```